drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/resources/org/drools/workbench/screens/scenariosimulation/client/resources/i18n/ScenarioSimulationEditorConstants.properties

```diff
@@ -130,5 +130,6 @@ settings=Settings
 uploadWarning=Make sure the headers in the selected CSV file have not been modified. Otherwise, the spreadsheet may not be successfully imported.
 importFailedMessage=A CSV file with an incorrect header layout has been imported. Make sure the imported CSV file has unmodified headers.
 importErrorTitle=Error during import
+running=Running
```

Comment:
Please change this to `Running...` to be consistent with other loading indicators.

Reply:
Done

Reply:
Thanks!